### PR TITLE
Error loading volume no app properties

### PR DESCRIPTION
**Pull request by akellapavani**

### CODE REVIEW

File: charts/bandstand-web-service/templates/_deployment.yaml [hunk 1]
```diff
@@ -95,12 +95,14 @@ spec:
               drop:
                 - all
       volumes:
+        { { - if or .Values.properties .Values.envProperties } }
         - name: config
           configMap:
```
Comment (vinay-thakkar) [inline charts/bandstand-web-service/templates/_deployment.yaml:100]: Would it not fail on line 42 for `volumeMount` ? if the volume `config` isn't defined ?

Comment (gavinclarkeuk) [reply charts/bandstand-web-service/templates/_deployment.yaml:100]: Yes, that needs wrapping in the same if statement

File: charts/bandstand-web-service/templates/_deployment.yaml [hunk 2]
```diff
@@ -95,12 +95,14 @@ spec:
...
             name: {{ $relName }}
             items:
               - key: "app.properties"
                 path: "app.properties"
+        { { - end } }
```
Comment (gavinclarkeuk) [inline charts/bandstand-web-service/templates/_deployment.yaml:105]:
```suggestion
        {{ - end }}
```

File: charts/bandstand-web-service/templates/_deployment.yaml [hunk 3]
```diff
@@ -95,12 +95,14 @@ spec:
               drop:
                 - all
       volumes:
+        { { - if or .Values.properties .Values.envProperties } }
```
Comment (gavinclarkeuk) [inline charts/bandstand-web-service/templates/_deployment.yaml:98]:
```suggestion
        {{ - if or .Values.properties .Values.envProperties }}
```

File: charts/bandstand-web-service/templates/_deployment.yaml (hunk 1 above)
Comment (akellapavani) [reply charts/bandstand-web-service/templates/_deployment.yaml:102]: added  at 42

**Review by gavinclarkeuk — APPROVED** [review]

**Pull request merged**